frontend/public/components/overview/index.tsx

```diff
@@ -231,6 +231,51 @@ const getResourcePausedAlert = (resource: K8sResourceKind): OverviewItemAlerts =
   };
 };
 
+const getBuildAlerts = (buildConfigs: K8sResourceKind[]): OverviewItemAlerts => {
+  const buildAlerts = {};
+  const addAlert = (build: K8sResourceKind, buildPhase: string) => _.set(buildAlerts, `${build.metadata.uid}--${buildPhase}`, {severity: buildPhase, message: _.get(build, ['status', 'message'], buildPhase)});
+
+  _.each(buildConfigs, bc => {
+    let seenComplete = false;
+    // Requires builds to be sorted by most recent first.
+    _.each(_.get(bc, 'builds'), (build: K8sResourceKind) => {
+      const buildPhase = _.get(build, ['status', 'phase']);
```

Comment:
@spadgett, if I remove the `_.get`, I get the previously mentioned TypeScript error.  Changing `(buildConfigs: K8sResourceKind[])` to `(buildConfigs: BuildConfigOverviewItem[])` does not fix it and causes compilation errors.  Please advise.